lib/usercopy.c

```diff
@@ -13,6 +14,7 @@ unsigned long _copy_from_user(void *to, const void __user *from, unsigned long n
 	if (likely(access_ok(from, n))) {
 		instrument_copy_from_user(to, from, n);
 		res = raw_copy_from_user(to, from, n);
+		add_address(from, n, _RET_IP_, to);
```

Comment:
Please fix the size.

Reply:
you mean sending res instead of n to add_address right?